File: android-app/baselibrary/src/main/java/com/musketeer/baselibrary/Activity/BaseFragmentUITask.java
package com.musketeer.baselibrary.Activity;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.ViewGroup;

/**
 * Created by zhongxuqi on 15-10-25.
 */
public interface BaseFragmentUITask {
    /**
     * 设置界面
     */
    void setContentView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState);

    /**
     * 初始化视图
     */
    void initView();

    /**
     * 初始化数据
     */
    void initEvent();

    /**
     * 初始化数据
     */
    void initData();
}

File: android-app/app/src/main/kotlin/com/muskeeter/base/acitivity/BaseFragmentUITask.kt
package com.muskeeter.base.acitivity

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

/**
 * Created by zhongxuqi on 08/07/2017.
 */
interface BaseFragmentUITask {

    /**
     * 设置界面
     */
    fun setContentView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?)

    /**
     * 初始化视图
     */
    fun initView()

    /**
     * 初始化数据
     */
    fun initEvent()

    /**
     * 初始化数据
     */
    fun initData()
}